src/components/ContactPolicy/ContactPolicy.test.js

```diff
@@ -53,100 +50,316 @@ describe('ContactPolicy component', () => {
     );
   };
 
-  it('should show loading box while getting data', () => {
+  it('should show loading box while getting data', async () => {
     // Act
-    const { container } = render(<ContactPolicyComponent />);
-
+    render(<ContactPolicyComponent />);
     // Assert
-    const loadingBox = container.querySelector('.wrapper-loading');
-    expect(loadingBox).toBeInTheDocument();
-    waitFor(() => {
-      expect(loadingBox).not.toBeInTheDocument();
-    });
+    await waitForElementToBeRemoved(screen.getByTestId('wrapper-loading'));
   });
 
   it('should load data from api correctly', async () => {
     // Act
-    const { container } = render(<ContactPolicyComponent />);
+    render(<ContactPolicyComponent />);
 
     // Assert
-    await waitFor(() => {
-      expect(container.querySelector('input#contact-policy-switch')).not.toBeChecked();
-      expect(container.querySelector('input#contact-policy-input-amount').value).toEqual(
-        `${settingsDouble().emailsAmountByInterval}`,
-      );
-      expect(container.querySelector('input#contact-policy-input-interval').value).toEqual(
-        `${settingsDouble().intervalInDays}`,
-      );
-    });
+    await waitForElementToBeRemoved(screen.getByTestId('wrapper-loading'));
```

Comment:
So instead of having to wait, you now wait on the spinner to be removed, it's clearer.